compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -743,6 +744,11 @@ public void visit(BLangUserDefinedType userDefinedType) {
         /* ignore */
     }
 
+    @Override
+    public void visit(BLangTableQueryExpression tableQueryExpression) {
+
```

Comment:
Shall we add /* ignore */ here within the body

Reply:
added.